src/foam/u2/wizard/BaseWizardlet.js

```diff
@@ -279,9 +278,9 @@ foam.CLASS({
   ],
 
   listeners: [
-    function updateVisibilityFromSectionCount() {
-      if ( ! this.sections ) return;
-      this.atLeastOneSectionVisible_ = this.sections.filter(
+    function updateVisibilityFromSectionCount(sections) {
```

Comment:
I think this is caused by the listeners being called too early. A simpler solution, is to just revert the code and then add isFramed: true on line 280.5.

Reply:
I tried reverting everything back and adding isFramed: true, however in doing so, we get an unintended behaviour where a capability section that requires an auth check doesn't trigger the listener when its isAvailable slot value changes to true causing the wizard to not display said capability.

Reply:
Do you understand why this is?